backend/vcdat/app.py

```diff
@@ -1,9 +1,22 @@
-from flask import Flask
-app = Flask(__name__)
+from flask import Flask, send_from_directory
+import os
+app = Flask(__name__, static_url_path='')
 
 @app.route("/")
 def hello():
-    return "Hello, Climate"
+    path = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', 'frontend/src/'))
+    return send_from_directory(path, 'index.html')
+
+@app.route("/deps/<path:path>")
+def serve_resource_file(path):
+    dir_path = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', 'frontend/'))
```

Comment:
Also also, these will (eventually) be served by the web server, not Flask, so we'll want to wrap this with some kind of conditional based on whether it's being run as part of the flask server or not.